src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -46,7 +46,7 @@ The .NET Foundation licenses this file to you under the MIT license.
 
   <!-- Set up default feature switches -->
   <PropertyGroup>
-    <EnableUnsafeBinaryFormatterSerialization Condition="'$(EnableUnsafeBinaryFormatterSerialization)' == ''">false</EnableUnsafeBinaryFormatterSerialization>
+    <CustomResourceTypesSupport Condition="'$(CustomResourceTypesSupport)' == ''">false</CustomResourceTypesSupport>
```

Comment:
We do want to have `EnableUnsafeBinaryFormatterSerialization` to be off by default in NativeAOT. I think you want to keep the existing condition and add the condition for `CustomResourceTypesSupport` in addition to that.

Reply:
Both these (old and new) feature switches are disabled in [Microsoft.NET.ILLink.targets](https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L37-L39) now when `PublishTrimmed `is set to true. That now happens with the SDK fix, https://github.com/dotnet/sdk/pull/25927